main.go

```diff
@@ -21,6 +22,10 @@ import (
 
 var configFilePath string
 
+func init() {
+	broker.DefaultContextTimeout = 5 * time.Minute
```

Comment:
hackery. this should be configurable from broker-base

Reply:
successful hackery tho?

Reply:
do we need this init block in both main.go and sqs/provider.go? my impression is they both do the same thing (but maybe only sqs/provider.go applies during integration tests so we should keep that one?)

Reply:
yeah we shouldn't need both ... I thought I had removed from main already _removes_